lib/modules/accountSwitcher.js

```diff
@@ -14,12 +14,7 @@ export const module: Module<*> = new Module('accountSwitcher');
 
 module.moduleName = 'accountSwitcherName';
 module.category = 'myAccountCategory';
-module.description = `
```

Comment:
At the moment we aren't translating strings that have HTML inline due to it being a potential security concern. We still need to convert to markdown @erikdesjardins